services/common/src/components/forms/RenderField.tsx

```diff
@@ -44,6 +45,7 @@ const RenderField: FC<BaseInputProps> = ({
           >
             <>
               <Input
+                data-testid={input.name}
```

Comment:
I added this while trying to fix that odd test.  Decided to leave it, as it will allow any RenderField instances to be found in tests by using `getByTestId('{input.name}')`.